### DISCUSSION TITLE
How to setup proxy in Remotely app

### DISCUSSION THREAD

**Discussion by netroti128** [immense/Remotely#465] [Q&A]
Hi, I copied full file Example_Nginx_Config.txt to my nginx server, and what should I do next?
Setup proxy on my Debian? Im totally newbie in proxies...

**Comment by rcschaff82** [comment]
Copy the "server" block from the Example_Nginx_Config, and add it to a virtualhost in nginx.  Make sure you amend the servername variable, and port if you wish to change it.  Test the configuration with nginx -t.  If it passes, then reload nginx with "nginx -s reload"
